parsers/promotedTitle.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env nodejs
+var _ = require('lodash');
+var cheerio = require('cheerio');
+var moment = require('moment');
+var debug = require('debug')('promotedInfo');
+var parse = require('./lib/parse');
+var entities = require('entities');
+
+var postcount = 0;
+
+function tryPage(elem) {
+    try {
+        return elem.attr().onmouseover.replace(/.*\"http/, 'http').replace(/\".*/, '').replace(/\\/g, '');
+    } catch(error) {
+        return null;
+    };
+}
+function getPromotedTitle(snippet) {
+
+	var title;
+	var title_type;
+	var error = 0;
+	
+	var e_threshold;
+	var e_ptr;
+		
+	postcount ++;
+	
+    var $ = cheerio.load(snippet.html);
+    
+	var e_threshold = $('div.userContent');
+
+	e_ptr = ($(e_threshold).next().find(".clearfix"));
+	$(e_ptr).find("*").each(function(e) {
+		if (title == undefined) {
+			var re = /(div|a|span)/g;
```

Comment:
@nolash, I don't know if is the best way, I didn't see the htmls. but in general, try to avoid regexp, has been a problem for me in the past, and `cheerio` is loaded just to use at our best CSS3 selectors :)

also... 
https://cloud.githubusercontent.com/assets/89555/17433338/5de2ad0c-5b04-11e6-98eb-3a5ee1281ce9.png